test/unit/component/mxgraph/renderer/StyleComputer.test.ts

```diff
@@ -286,24 +290,48 @@ describe('Style Computer', () => {
     });
   });
 
-  describe('compute style - call activities', () => {
-    describe.each([
-      ['expanded', []],
-      ['collapsed', [ShapeBpmnMarkerKind.EXPAND]],
-    ])(`compute style - %s call activities`, (expandKind, markers: ShapeBpmnMarkerKind[]) => {
-      it(`${expandKind} call activity without label bounds`, () => {
-        const shape = newShape(newShapeBpmnCallActivity(markers), newLabel({ name: 'Arial' }));
-        const additionalMarkerStyle = markers.includes(ShapeBpmnMarkerKind.EXPAND) ? ';bpmn.markers=expand' : '';
-        const additionalTerminalStyle = !markers.includes(ShapeBpmnMarkerKind.EXPAND) ? ';verticalAlign=top' : '';
-        expect(computeStyle(shape)).toEqual(`callActivity${additionalMarkerStyle};fontFamily=Arial${additionalTerminalStyle}`);
+  describe('compute style - call activities calling process', () => {
```

Comment:
Don't rename the describe function as it now contains describe functions to split 'calling process' and 'global task' cases
```suggestion
  describe('compute style - call activities', () => {
```